learning/web/src/pages/cookies.js

```diff
@@ -0,0 +1,166 @@
+import React from "react"
+import Cookies from "universal-cookie"
+
+
+import Layout from "../components/Layout"
+import SEO from "../components/SEO"
+import styled from "styled-components"
+import { mhraWhite, primaryColor, mhra } from "../utils/colors"
+
+
+const StyledCookiePolicy = styled.section`
+  form {
+    label {
+      padding-top: 0;
+      display: inline-block;
+      padding: 40px 50px 40px 0;
+      input {
+        margin-right: 30px;
+      }
+    }
+
+    button {
+      appearance: none;
+      color: ${mhraWhite};
+      background-color: ${primaryColor};
+      border-radius: 5px;
+      border: 1px solid ${mhra};
+      display: block;
+      padding: 10px 20px;
+      cursor: pointer;
+    }
+  }
+`;
+
+
+
+class CookiePolicy extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      cookiesAllowed: true  // This will be updated once the page is loaded.
+    };
+  }
+  
+  componentDidMount() {
+    this.setState({
+      cookiesAllowed: window.localStorage.getItem("showCookieBanner") === "false"
+    });
+  }
+
+  handleCookieFormSubmit = (e) => {
+    e.preventDefault();
+
+    if (this.state.cookiesAllowed) {
+      window.localStorage.setItem("showCookieBanner", "false")
+    } else {
+      // Delete *everything*.
```

Comment:
Might be good to move this part to a separate function called e.g. `deleteAllStorage` or so, then this comment won't be needed.